src/main/java/onboarding/pages/android/MyCarinaDescriptionPage.java

```diff
@@ -41,8 +41,12 @@ public boolean isPageOpened(long timeout) {
     }
 
     @Override
-    public List<MenuItem> getMenuItems(){
+    public void openMenu(){
         leftMenu.click(TimeConstants.WAIT_INTERVAL);
```

Comment:
@DanyloZotov 
please use understandable constants. How long is it? 5 sec or 20? here normal 1-2sec